src/config/experiment.js

```diff
@@ -21,7 +22,15 @@ const getConfig = async (participantID, studyID) => {
       console.warn("Using default config")
     }
     renderer.send("save-config", experimentConfig, participantID, studyID)
-  } else {
+  }  else if (envConfig.USE_FIREBASE) {
+    const newConfig = await firestoreConfig(studyID, participantID);
+    if (newConfig) {
+      experimentConfig = newConfig;
```

Comment:
does this need to be a full config, or should it merge in with the defaults?

Reply:
I'm not entirely sure what merging with the default means @mcmcgrath13

Reply:
so instead of the newConfig entirely replacing the default config, it could instead only override things that it valued.

So if the default was
```json
{ "a": 1, "b": [5, 6, 3]}
```

You could have a config with just `{"a": 2}`

Then if you merged (`config = {...defaultConfig, ...newConfig}`), the result would be
```json
{ "a": 2, "b": [5, 6, 3]}
```

Right now, because it fully overrides, all keys must always be specified.